patches/minecraft/net/minecraft/block/BlockPane.java.patch

```diff
@@ -12,15 +12,25 @@
      }
  
      public Item func_180660_a(IBlockState p_180660_1_, Random p_180660_2_, int p_180660_3_)
-@@ -184,4 +187,11 @@
+@@ -184,4 +187,21 @@
      {
          return new BlockStateContainer(this, new IProperty[] {field_176241_b, field_176242_M, field_176244_O, field_176243_N});
      }
 +
-+    public boolean canPaneConnectTo(IBlockAccess world, BlockPos pos, EnumFacing dir)
++    /* ======================================== FORGE START ======================================== */
++
++    @Override
++    public boolean canBeConnectedTo(IBlockAccess world, BlockPos pos, EnumFacing facing)
 +    {
-+        BlockPos off = pos.func_177972_a(dir);
-+        IBlockState state = world.func_180495_p(off);
-+        return func_150098_a(state.func_177230_c()) || state.isSideSolid(world, off, dir.func_176734_d());
++        Block connector = world.func_180495_p(pos.func_177972_a(facing)).func_177230_c();
++        return connector instanceof BlockPane;
 +    }
++
++    public boolean canPaneConnectTo(IBlockAccess world, BlockPos pos, EnumFacing facing)
```

Comment:
do not rename the parameters here, it should match the old ones:
`public boolean canPaneConnectTo(IBlockAccess world, BlockPos pos, EnumFacing dir)`